pkg/metrics/metrics.go

```diff
@@ -735,6 +749,40 @@ func UpdateDatabaseQueryDurationMetric(status string, queryType string, elapsed
 
 // #### Metrics for Database - End ####
 
+// create a new gaugeVec for the prewarming status info count per cluster_id, instance_type and status.
+// The metric contains the timestamp (in milliseconds) information indicating the time the metric was reported.
+// The value of the timestamp is an increasing value.
+var prewarmingStatusInfoCountMetric = prometheus.NewGaugeVec(
+	prometheus.GaugeOpts{
+		Subsystem: KasFleetManager,
+		Name:      PrewarmingStatusInfoCount,
+		Help:      "count of reserved kafka instances in a given status thats deployed on the given cluster_id. The metric contains the timestamp (in milliseconds) information indicating the time the metric was reported.",
```

Comment:
`thats` can be removed

Reply:
The part about timestamp doesn't apply right now no?

Reply:
Good catch! The timestamp part doesn't apply. I was in middway of adding it but then decided to keep it for later phases implementation. I'll remove it from the description.

Reply:
Done